central_v2/src/components/EGHeader.tsx

```diff
@@ -0,0 +1,240 @@
+import React, { useState } from 'react';
+import { useTheme, styled } from '@mui/material/styles';
+import { Box, Button, Typography, Collapse, IconButton } from '@mui/material';
+import rightonlogo from '../images/rightonlogo.svg';
+import dice from '../images/Dice.svg';
+import qmarks from '../images/qmarks.svg';
+import books from '../images/books.svg';
+import profile from '../images/profileplaceholder.svg';
+import hamburger from '../images/hamburger.svg';
+import hamburgerX from '../images/hamburgerX.svg';
+import plus from '../images/plus.svg';
+import { ScreenSize } from '../lib/HostModels';
+import { Screen } from '../lib/ScreenEnums';
+
+interface EGHeaderProps {
+  screenSize: ScreenSize;
+  isXLScreen: boolean;
+  onScreenChange: (newScreen: Screen) => void;
+  menuOpen: boolean;
+  setMenuOpen: (menuOpen: boolean) => void;
+}
+
+interface EGHeaderContainerProps {
+  screenSize: ScreenSize;
+  menuOpen: boolean;
+}
+const EGHeaderContainer = styled(Box)<EGHeaderContainerProps>(({ screenSize, menuOpen }) => ({
+  height: screenSize === ScreenSize.SMALL ? '77px' : '94px',
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  width: '100%',
+  padding: '16px 32px 16px 32px',
+  boxSizing: 'border-box',
+  position: 'sticky',
+  top: 0,
+  zIndex: 1000
```

Comment:
Can we get these zIndex's down to 0-10? 

So like 0 is default, and then just increase by 1 to stack them? Makes it easier to read what is on top of what